translations/texts/objects/apex/apexbrainjar/apexbrainjar.object.json

```
@@ -19,7 +19,8 @@
       ]
     },
     "Texts": {
-      "Eng": "A brain in a jar? I ain't comfortable with that."
+      "Eng": "A brain in a jar? I ain't comfortable with that.",
+      "Rus": "Мозг в банке? Мне это неудобно."
```

```suggestion
      "Rus": "Мозг в банке? Что-то мне не по себе."
```
Речь идёт о том, что говорящему неприятно находиться рядом с ним, а не о том, что ему неудобно держать свой мозг в банке